src/gui/widgets/symbol_render_widget.cpp

```diff
@@ -834,12 +834,25 @@ void SymbolRenderWidget::deleteSymbols()
 	}
 	
 	// delete symbols in order
+	bool yes_to_all = false;
 	for (auto* symbol : saved_selection)
 	{
-		if (map->existsObjectWithSymbol(symbol))
+		if (map->existsObjectWithSymbol(symbol) && !yes_to_all)
```

Comment:
I would suggest to put the cheap boolean test first. No need to walk through the objects with yes_to_all.

Reply:
Sure. Will include both changes in the next iteration.